### DISCUSSION TITLE
大佬帮帮忙

### DISCUSSION THREAD

**Discussion by eltPlay** [Yin-Hongwei/vue-spring-music#169] [Q&A] [answered]
服务器可以运行但网站页面图片显示不出来，请问有什么解决办法么？

**Comment by Yin-Hongwei** [comment, marked as answer]
> 服务器可以运行但网站页面图片显示不出来，请问有什么解决办法么？

看看这里 https://github.com/Yin-Hongwei/music-website/issues/168
